src/destination/selector.js

```diff
@@ -69,6 +69,16 @@ export default function destinationSelectorFactory($container, config) {
                 if (this.resourceSelector) {
                     this.resourceSelector.update(results, params);
                 }
+            },
+
+            /**
+             * Updates the url on taskCreationButton
+             * @param {String} url - url of the task creation
+             */
+            updateTaskCreationUrl: function updateTaskCreationUrl(url) {
+                if (this.taskCreationButton && this.taskCreationButton.config) {
+                    this.taskCreationButton.config.taskCreationUrl = url;
```

Comment:
I think we should also update `this.config.taskCreationUrl` with the new value, to avoid being out of sync.

Reply:
good idea (however ultimately it is not really used)